handbook/product/user_research/user_research_participant.md

```diff
@@ -0,0 +1,18 @@
+# So you’re about to participate in user testing
+
+Thank you so much for helping with our user research! Remote moderated hallway testing is a great way for us to quickly test and iterate on our ideas so that we can identify opportunities and challenges early.
+Here’s everything you need to know ahead of time.
+
+## Prerequisites
+
+- The tool we use, Lookback, works with Chrome. You’ll need to use Chrome to join the session.
+- Please make sure you install the [Participate by Lookback](https://chrome.google.com/webstore/detail/participate-by-lookback/ppapgcbnefafdghpfglgilaghielefgn/related) extension for Chrome.
+- Before joining the session, please make sure to turn on “Do not disturb” and hide any windows you don’t want others to see.
+
+## What to expect
+
+1. You will receive a link to join the session ahead of time. You can join anytime before the session begins. Whoever’s running the session will see you’ve joined and will start the session when they’re ready.
+2. The session will be recorded (audio, video, and screen) so we can refer to it in the future. We won’t share it outside the team.
+3. You can leave at any time, ask questions at any time, and share as much or as little information as you’d like.
+4. While you can ask questions at any time, we might not be able to answer! Often, we’re interested in seeing how you answer that question for yourself.
+5. We’re testing our ideas, **not you**! You can’t do anything wrong.
```

Comment:
```suggestion
5. We’re testing our ideas, **not you**! You can’t do anything wrong.
6. There is nothing you have to do after the test. To keep you in the loop and satisfy your curiosity, the designer will share the final solution with you, once it's ready!
```

I like to follow up with participants, just to show them how they've influenced the results and keep them excited about testing in thefuture.